conf/embl_hd.config

```
@@ -0,0 +1,46 @@
+params {
+    config_profile_contact     = "gitlab-data-sciences-sci-workflows-nextflow-config-6076-issue-@embl.de"
+    config_profile_description = "The EMBL Heidelberg HPC cluster profile"
+    config_profile_name        = 'EMBL_HD'
+    config_profile_url         = 'https://git.embl.de/data-sciences/sci-workflows/nextflow-config'
+}
+
+apptainer {
+    enabled = true
+    autoMounts = true
+    docker.enabled = false
+    pullTimeout = "3 hours"                // the default is 20 minutes and fails with large images
+    envWhitelist = 'CUDA_VISIBLE_DEVICES'  // allow the bounding of GPU visible device variable into the containers
+}
+
+process {
+    scratch = '$SCRATCHDIR'
+    maxRetries = 3
+    errorStrategy = { task.attempt<3 ? 'retry' : 'ignore' }
```

```suggestion
    errorStrategy = { task.attempt <3 ? 'retry' : 'ignore' }
```
Are you sure you want to set this globally? This means pipelines will never actually fail if one of the steps breaks quite badly, it will always report the pipeline finished as expected.